### PR TITLE
chore: Install sept 2021 security patches in Windows VHDs

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [x] No
- [ ] Yes

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by marosset** [comment]
@jackfrancis @mboersma Can you give this a review?

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -199,9 +199,9 @@ function Install-WindowsPatches {
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
             # Find a specific patch at https://www.catalog.update.microsoft.com/Search.aspx?q=<KB-number>
-            # Cumulative updates for WS 2019 from July 13, 2021
-            # https://www.catalog.update.microsoft.com/Search.aspx?q=KB5004244
-            $patchUrls = @("http://download.windowsupdate.com/d/msdownload/update/software/secu/2021/07/windows10.0-kb5004244-x64_5685623313a6de061e0c42fed3391c29750a6b1b.msu")
+            # Cumulative updates for WS 2019 from Setp 14, 2021
+            # https://www.catalog.update.microsoft.com/Search.aspx?q=KB5005568
+            $patchUrls = @()
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1:204]: so this means that for this image version we don't add any targeted patches?

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1:204]: Correct, this time the VHD we are starting with already containers the Sept 2021 patches.
Normally it takes a few days for the base marketplace images to become available to we start with a base image from the previous month and apply this months patches.
For this month the base image was already available by the time we authored the PR.

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Review by mboersma — APPROVED** [review]
/lgtm

**Pull request merged**